lib/src/scss-shared/_nx-tag-variables.scss

```diff
@@ -85,17 +89,20 @@
 
 .nx-selectable-color--yellow {
   @extend %nx-tag-hover-style;
-
+  --nx-color-tag-selected-hover-background: var(--nx-swatch-yellow-55);
+  --nx-color-tag-unselected-hover-background: var(--nx-swatch-yellow-90);
+  
   @include nx-dark-mode-helpers.dark-mode {
     --nx-color-tag-selected-hover-background: var(--nx-swatch-yellow-60);
     --nx-color-tag-unselected-hover-background: var(--nx-swatch-yellow-80);
   }
-
 }
 
 .nx-selectable-color--lime, // deprecated
 .nx-selectable-color--kiwi {
   @extend %nx-tag-hover-style;
+  --nx-color-tag-selected-hover-background: var(--nx-swatch-green-30);
+  --nx-color-tag-unselected-hover-background: var(--nx-swatch-green-85);
```

Comment:
> It looks like this one is supposed to be green-90

I double checked with Jason on this one (since the hover/focus state is listed as green-85), and it is indeed supposed to be green-85. Thanks for catching that one!